### PR TITLE
Translation ja_JP.ini update

### PR DESCRIPTION
**Pull request by nassau-tk**
Translation to Japanese on menu.

### CODE REVIEW

File: assets/lang/ja_JP.ini [hunk 1]
```diff
@@ -1224,17 +1224,3 @@ Details = Details
...
-Distance to 2D menus and scenes = Distance to 2D menus and scenes
-Field of view scale = Field of view scale
-Force 72Hz update = Force 72Hz update
-Heads-up display scale = Heads-up display scale
-Map controller movements to keys = Map controller movements to keys
-Motion needed to generate action = Motion needed to generate action
-Stereoscopic vision (Experimental) = Stereoscopic vision (Experimental)
-Virtual reality = Virtual reality
-VR camera = VR camera
-VR controllers = VR controllers
```
Comment (hrydgard) [inline assets/lang/ja_JP.ini:1240]: Looks like this section got lost? Please add it back, even if you're not gonna translate it right now (it's only visible for VR users)

Comment (nassau-tk) [reply assets/lang/ja_JP.ini:1240]: OK
I made from v1.13.2-2456.
And, merge and tranlate it from v1.13.2-2464.

And....How to upload it???

Comment (nassau-tk) [reply assets/lang/ja_JP.ini]: I edited the file on here.
Is it OK??

**Comment by hrydgard** [comment]
all good, thanks!

**Pull request merged**